messages/main.proto

```diff
@@ -793,3 +794,17 @@ message ShutdownScheduled
     ShutdownReason shutdown_reason = 1;
     optional uint32 ms_until_shutdown = 2;
 }
+
+message Polarizer
+{
+    enum Command {
+        // perform homing procedure which last a few seconds
```

Comment:
can we also support custom angle command? Or it would require too many changes?

Reply:
done in the message, testing the implementation